projects/rocprofiler-systems/source/lib/core/config.cpp

```diff
@@ -293,20 +293,21 @@ configure_settings(bool _init)
         get_env<size_t>("ROCPROFSYS_NUM_THREADS", 1), "threading", "performance",
         "sampling", "parallelism", "advanced");
 
-    ROCPROFSYS_CONFIG_SETTING(bool, "ROCPROFSYS_TRACE", "Enable perfetto backend",
-                              _default_perfetto_v, "backend", "perfetto");
+    ROCPROFSYS_CONFIG_SETTING(bool, "ROCPROFSYS_TRACE_LEGACY", "Enable perfetto backend",
+                              false, "backend", "perfetto");
```

Comment:
Maybe something like this? Just wanted to change the message emphasis that this is the old model.
Do we need to guard against using both options?

```suggestion
    ROCPROFSYS_CONFIG_SETTING(bool, "ROCPROFSYS_TRACE_LEGACY", "Enable legacy perfetto backend, "
                              "rather than the new caching model. Should not be used at the same time as ROCPROFSYS_TRACE.",
                              false, "backend", "perfetto");
```